OMCompiler/Compiler/Script/CevalScript.mo

```diff
@@ -1924,9 +1924,10 @@ protected function errorLevelToValue
   output Values.Value val;
 algorithm
   val := match severity
-    case ErrorTypes.ERROR() then makeErrorEnumLiteral("ErrorLevel","error",1);
-    case ErrorTypes.WARNING() then makeErrorEnumLiteral("ErrorLevel","warning",2);
-    case ErrorTypes.NOTIFICATION() then makeErrorEnumLiteral("ErrorLevel","notification",3);
+    case ErrorTypes.INTERNAL() then makeErrorEnumLiteral("ErrorLevel","internal",1);
+    case ErrorTypes.ERROR() then makeErrorEnumLiteral("ErrorLevel","error",2);
+    case ErrorTypes.WARNING() then makeErrorEnumLiteral("ErrorLevel","warning",3);
+    case ErrorTypes.NOTIFICATION() then makeErrorEnumLiteral("ErrorLevel","notification",4);
```

Comment:
Would it not have made sense to make `INTERNAL()` as the fourth element to keep the old error/warning/notification have the same values?

Reply:
I wanted to keep the same order as in ErrorTypes.Severity. I don't think anything uses the actual value of the enumeration anyway, the enumeration seems to only be used by the API which just uses the names.